docs/_documentations/remote-deploying-codewind.md

```diff
@@ -122,7 +161,7 @@ devex001   jane       codewind    119m   devex001   Complete       https://codew
 
 ## 5. Removing a Codewind instance
```

Comment:
(I know that this is nothing to do with this PR) IMO I'd put `## 5. Removing a Codewind instance` under `## Next steps` so that the user reads next steps after getting Codewind up and running.
I'd also remove the number from `## 5. Removing a Codewind instance` as it hints that that is the next step (and we don't want them to ever do that)

Reply:
Moved `removing` to Next steps, and step #5 is no more.